test/Chirp.Infrastructure.Tests/CheepRepositoryUnitTests.cs

```diff
@@ -31,10 +34,43 @@ public CheepRepositoryUnitTests()
 		DbInitializer.SeedDatabase(context);
 
 		_cheepRepository = new CheepRepository(context);
+		_context = context;
 	}
 
 	private protected void Dispose() => _connection.Dispose();
 
+	[Fact]
+	public void CreateNewCheep_FromNonExistingAuthor_ThrowsException()
+	{
+		// Arrange
+		string email = "email@domain.com";
+		string name = "name";
+		string cheepMessage = "Message";
+
+		CreateCheepDTO cheep = new CreateCheepDTO()
+		{
+			CheepGuid = new Guid(),
+			Text = cheepMessage,
+			Name = name,
+			Email = email
+		};
+
+		Exception? exception = null;
+
```

Comment:
Many great things seems to be accomplished in this pull request. Maybe in CreateNewCheep_FromNonExistingAuthor_ThrowsException() test, we should not create an exception with null value, and instead move it and the Assert-functions into a "finally"-scope